packages/commerce-sdk-react/src/index.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2022, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import CommerceAPIProvider from './provider'
+import useCommerceApi from './hooks/useCommerceApi'
+export * from './hooks/types'
+export * from './hooks'
```

Comment:
If I want to import and use a hook in a template, currently it looks like I can't import from just `commerce-react-sdk`. I had to specify a relative path. Is there any webpack config that needs to be updated perhaps?

Reply:
Sorry, that's my bad. I forgot to install the `commerce-sdk-react` dependency first.